kubernetes/src/main/java/io/kubernetes/client/ApiClient.java

```diff
@@ -73,6 +73,10 @@
 
     private HttpLoggingInterceptor loggingInterceptor;
 
+    public static final String PATCH_FORMAT_JSON_PATCH = "application/json-patch+json";
+    public static final String PATCH_FORMAT_JSON_MERGE_PATCH = "application/merge-patch+json";
+    public static final String PATCH_FORMAT_STRATEGIC_MERGE_PATCH = "application/strategic-merge-patch+json";
+
```

Comment:
Shouldn't apiclient be not modified as its generated by swagger.

Reply:
it was already modified and skipped by setting rule here

[https://github.com/kubernetes-client/java/blob/2e43cbe52c2379755653f18a29957f3896a6722e/kubernetes/.swagger-codegen-ignore#L11](https://github.com/kubernetes-client/java/blob/2e43cbe52c2379755653f18a29957f3896a6722e/kubernetes/.swagger-codegen-ignore#L11)

Reply:
Yeah, we really shouldn't, but there were a bunch of fixes we needed...

We should try regenerating in case there are bug fixes we could pick up...

For now, this is fine.